infra/core/managed-cluster/aks-cluster.bicep

```diff
@@ -54,6 +59,48 @@ module managedIdentity 'br/SharedDefraRegistry:managed-identity.user-assigned-id
   }
 }
 
+module managedIdentityAso 'br/SharedDefraRegistry:managed-identity.user-assigned-identity:0.4.3' = {
+  name: 'aso-managed-identity-${deploymentDate}'
+  params: {
+    name: asoPlatformManagedIdentity
+    tags: union(tags, tagsAsoMi)
+    location: location
+    lock: 'CanNotDelete'
+    federatedIdentityCredentials: [
+      {
+        name: '${asoPlatformManagedIdentity}-fic'
```

Comment:
provide example @ajaysoni-cap

Reply:
@DaRoszko Discussed with @ajaysoni-cap and agreed to take off the '-fic' so the Federated Credential name matches the MI name.